beginners-yoga-course.html

```diff
@@ -0,0 +1,79 @@
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Prema Healing Cleveland</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Signika:300">
+  <link rel="stylesheet" href="main.css">
+  <style type="text/css"></style>
+</head>
+<body>
+<div class="pageWrap">
+
+<!-- HEADER -->
+<div class="gridCell--row gridCell--padded gridCell--row--justifyCenter">
+  <header class="pageHeader">
+    <h1 class="headline pageHeader__headline align--center">
+      <span class="headline__line pageHeader__primary inverted">Prema Healing</span>
+      <span class="headline__line pageHeader__secondary">Cleveland</span>
+    </h1>
+  </header>
+</div>
```

Comment:
I'd also note that on narrower screens, the header isn't centered on the page which looks strange against the centered content below. On the home page it wasn't as weird because all of the text was left aligned and it used to say "Prema Yoga Cleveland" which aligned nicely. So you might consider reworking the header to align better (by shrinking "Prema Healing" or enlarging "Cleveland" or something else).

Looks weird to me:
![image](https://cloud.githubusercontent.com/assets/963631/13287100/35fb9e4a-dad3-11e5-95a1-62d3a50e8d23.png)

Looks fine to me:
![image](https://cloud.githubusercontent.com/assets/963631/13287107/422e6f1c-dad3-11e5-9490-0e3d5a825d4d.png)

Looks very awkward to me:
![image](https://cloud.githubusercontent.com/assets/963631/13287115/4c18b8b6-dad3-11e5-9fc6-96eb378274e7.png)